### PR TITLE
GAWB-2526 adding missing tests for GET /api/groups/{groupName}

### PR DESCRIPTION
**Pull request by gpolumbo-broad**
\<your comments for this PR go here\>

Have you read [CONTRIBUTING.md](../CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [ ] I've followed [the instructions](https://github.com/broadinstitute/sam/blob/develop/CONTRIBUTING.md#api-changes) if I've made any changes to the API, _especially_ if they're breaking changes
- [ ] I've updated the RC_XXX release ticket with any manual steps required to release this change
- [ ] I've updated the [FISMA documentation](https://github.com/broadinstitute/sam/blob/develop/CONTRIBUTING.md#fisma-documentation-changes) if I've made any security-related changes, including auth, encryption, or auditing

In all cases:

- [ ] Get two thumbsworth of review and PO signoff if necessary
- [ ] Verify all tests go green
- [ ] Squash and merge; you can delete your branch after this **unless it's for a hotfix**. In that case, don't delete it!
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: src/test/scala/org/broadinstitute/dsde/workbench/sam/api/ManagedGroupRoutesSpec.scala [hunk 1]
```diff
@@ -66,6 +66,49 @@ class ManagedGroupRoutesSpec extends FlatSpec with Matchers with ScalatestRouteT
...
+    val samRoutes = TestSamRoutes(resourceTypes)
+    assertCreateGroup(samRoutes)
+
+    val newGuyEmail = WorkbenchEmail("newGuy@organization.org")
+    val newGuy = UserInfo(OAuth2BearerToken("newToken"), WorkbenchUserId("NewGuy"), newGuyEmail, 0)
+    val newGuyRoutes = new TestSamRoutes(samRoutes.resourceService, samRoutes.userService, samRoutes.statusService, samRoutes.managedGroupService, newGuy, samRoutes.mockDirectoryDao)
+    assertCreateUser(newGuyRoutes)
+
+    Get(s"/api/group/$groupId") ~> newGuyRoutes.route ~> check {
+      status shouldEqual StatusCodes.OK
```
Comment (breilly2) [inline src/test/scala/org/broadinstitute/dsde/workbench/sam/api/ManagedGroupRoutesSpec.scala:108]: Shouldn't this be `NotFound`?

Comment (gpolumbo-broad) [reply src/test/scala/org/broadinstitute/dsde/workbench/sam/api/ManagedGroupRoutesSpec.scala:108]: Actually, I think the test description was incorrect.  IIRC last week, we do want this to return OK.  @dvoet can you please confirm?

**Review by breilly2 — APPROVED** [review]
As long as 200 is the right response for the case I commented on, 👍

File: src/test/scala/org/broadinstitute/dsde/workbench/sam/api/ManagedGroupRoutesSpec.scala (hunk 1 above)
Comment (dvoet) [reply src/test/scala/org/broadinstitute/dsde/workbench/sam/api/ManagedGroupRoutesSpec.scala:108]: OK is ok

**Review by dvoet — APPROVED** [review]

**Pull request merged**